yark/archive.py

```diff
@@ -352,6 +352,23 @@ def search(self, id: str):
         # Raise exception if it's not found
         raise VideoNotFoundException(f"Couldn't find {id} inside archive")
 
+    def search_livestreams(self, id: str):
+        """Searches archive for a livestream with the corresponding `id` and returns"""
+        # Search
+        for video in self.livestreams:
+            if video.id == id:
+                return video
```

Comment:
This should have the `raise VideoNotFoundException` like the videos and shorts. Would be best making a general `_search(self, id: str, videos: Videos)` and these three methods use that instead